pkg/ibmcsidriver/ibm_csi_driver.go

```diff
@@ -24,11 +24,12 @@ import (
 
 	commonError "github.com/IBM/ibm-csi-common/pkg/messages"
 	nodeMetadata "github.com/IBM/ibm-csi-common/pkg/metadata"
+	mountManager "github.com/IBM/ibm-csi-common/pkg/mountmanager"
 	"github.com/IBM/ibm-csi-common/pkg/utils"
 	cloudProvider "github.com/IBM/ibmcloud-volume-file-vpc/ibmcloudprovider"
 	csi "github.com/container-storage-interface/spec/lib/go/csi"
 	"go.uber.org/zap"
-	"k8s.io/kubernetes/pkg/util/mount"
+	//mount "k8s.io/mount-utils"
```

Comment:
Is this required ? can we remove commented code ?

Reply:
Removed